Libplanet.Net.Tests/Consensus/Context/ContextProposerTest.cs

```diff
@@ -230,7 +230,8 @@ public async void EnterPreVoteNil()
             var stepChangedToPreVote = new AsyncAutoResetEvent();
             var nilPreVoteSent = new AsyncAutoResetEvent();
             var (_, context) = TestUtils.CreateDummyContext(
-                height: 5); // Peer1 should be a proposer
+                height: 5,
+                validatorSet: Libplanet.Tests.TestUtils.ValidatorSet); // Peer1 should be a proposer
```

Comment:
```suggestion
                height: 5); // Peer1 should be a proposer
```
Seems like original code would work properly.